pkg/kubectl/cmd/version.go

```diff
@@ -34,7 +34,7 @@ func NewCmdVersion(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			cmdutil.CheckErr(err)
 		},
 	}
-	cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
```

Comment:
Per [this comment](https://github.com/kubernetes/kubernetes/issues/15471#issuecomment-147500443), please leave the shorthand flag in and instead add
```
cmd.Flags().MarkShorthandDeprecated("client", "please use --client instead")
```